datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -391,6 +393,13 @@ private static void configureDatasetResolvers(final RuntimeWiring.Builder builde
                         (env) -> ((SearchResult) env.getSource()).getEntity()))
                 ) 
             )
+            .type("BrowseResults", typeWiring -> typeWiring
+                .dataFetcher("entities", new AuthenticatedResolver<>(
+                    new EntityTypeBatchResolver(
+                        ENTITY_TYPES.stream().collect(Collectors.toList()),
+                        (env) -> ((BrowseResults) env.getSource()).getEntities()))
+                )
+            )
```

Comment:
side note- we should really put these two in a configureSearchAndBrowseResolvers...